src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Random randomsNumber = new Random();
+        int numberOfRandBall = randomsNumber.nextInt(100);
+        ColorSupplier randomsColor = new ColorSupplier();
+        String colorOfRandomBall = randomsColor.getRandomColor();
+        Ball randomsBall = new Ball(colorOfRandomBall, numberOfRandBall);
```

Comment:
use constant for 100
```suggestion
        Random random = new Random();
        int randomNumber = random.nextInt(100);
        ColorSupplier colorSupplier = new ColorSupplier();
        String randomColor = colorSupplier.getRandomColor();
        Ball randomBall = new Ball(randomColor, randomNumber);
```